assets/js/src/subscribers/form.jsx

```diff
@@ -39,6 +39,7 @@ const fields = [
       unconfirmed: MailPoet.I18n.t('unconfirmed'),
       unsubscribed: MailPoet.I18n.t('unsubscribed'),
       bounced: MailPoet.I18n.t('bounced'),
+      inactive: MailPoet.I18n.t('inactive'),
```

Comment:
Just a small detail. [Spec 3 a)](https://mailpoet.atlassian.net/browse/MAILPOET-1973) says that inactive should after unsubscribed.